lib/rucio/daemons/conveyor/common.py

```diff
@@ -101,6 +104,9 @@ def submit_transfer(external_host, job, submitter='submitter', logging_prepend_s
         record_timer('daemons.conveyor.%s.submit_bulk_transfer.per_file' % submitter, (time.time() - start_time) * 1000 / len(job['files']))
         record_counter('daemons.conveyor.%s.submit_bulk_transfer' % submitter, len(job['files']))
         record_timer('daemons.conveyor.%s.submit_bulk_transfer.files' % submitter, len(job['files']))
+    except (TransferToolTimeout, TransferToolWrongAnswer) as error:
+        logging.error(prepend_str + str(error))
```

Comment:
is it fine to `continue` here?

Reply:
I think it is, as the `eid` will be `None` and this is handled correctly later on and setting the request to `RequestState.SUBMISSION_FAILED`